packages/proto-signing/src/registry.ts

```diff
@@ -95,13 +95,14 @@ export class Registry {
    * change the `customTypes` argument to override the default types if set.
    * See https://github.com/cosmos/cosmjs/issues/994
    */
-  public constructor(customTypes: Iterable<[string, GeneratedType]> = []) {
+  public constructor(customTypes?: Iterable<[string, GeneratedType]>) {
```

Comment:
Can you read through the doc comment and update the parts that change with this PR?